core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3648,6 +3648,33 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_CLIENT_CACHE_TIMEOUT_DURATION =
+      new Builder(Name.USER_CLIENT_CACHE_TIMEOUT_DURATION)
+          .setDefaultValue("60sec")
+          .setDescription("The timeout duration for local I/O operations on "
+              + "reading/writing/deleting in the cache. Local cache operations that time out will "
+              + "fail and fallback to external file system.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.CLIENT)
+          .build();
+  public static final PropertyKey USER_CLIENT_CACHE_TIMEOUT_ENABLED =
+      new Builder(Name.USER_CLIENT_CACHE_TIMEOUT_ENABLED)
```

Comment:
would it be easier to just have a timeout of `-1` represent disabled, instead of 2 consistent parameters?

Reply:
that's my original way . but it was not a good practise as you need to be very careful to call
`conf.getMs(PropertyKey.USER_CLIENT_CACHE_TIMEOUT_DURATION)` which will throw Exception. 
this will complicate the source code. IN this case, i prefer a bit more verbose but make sure the produciton
code is more type-safe

Reply:
Why will it throw an exception? We do this for sync interval: https://github.com/Alluxio/alluxio/blob/master/core/common/src/test/java/alluxio/conf/InstancedConfigurationTest.java#L395

Reply:
ah, sounds good